_posts/2017-01-25-docker-volumes.markdown

```diff
@@ -328,7 +328,7 @@ docker container run -v HOST_PATH:CONTAINER_PATH [OPTIONS] IMAGE [CMD]
 Note: HOST_PATH and CONTAINER_PATH can be a folder or file. HOST_PATH must exist before running this command.
 
 Several cases to consider:
-- the CONTAINER_PATH does not exist within the container
```

Comment:
I believe this case is correct as it is.. In this particular case, the CONTAINER_PATH doesn't exist beforehand in the container